cannonfile.toml

```diff
@@ -0,0 +1,67 @@
+name = 'cartesi-rollups'
+description = 'Smart Contracts for Cartesi Rollups'
+version = '2.0.0'
+
+[deploy.InputBox]
+artifact = 'InputBox'
+create2 = true
```

Comment:
should we still use the safe-singleton-factory, or are there no more reasons?

I remember we switched to safe-singleton-factory was because it has much more networks supported, and back in time we had a more ambitious plan in terms of network support.

Reply:
There are pros and cons to using Safe.

Pros:

- Can be deployed to networks that don't support legacy transactions

Cons:

- Not deployed by default on Anvil and Cannon local networks
- Depends on Safe team to sign transactions

Since we only support networks that support legacy transactions, I'd say there are no more pros to using Safe over Archnid.